addons/project/migrations/9.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,187 @@
+NEW ir.rule: project.report_project_task_user_report_comp_rule
+# New rules ---> Nothing to do
+
+NEW ir.ui.menu: base.menu_project_general_settings
+NEW ir.ui.menu: project.menu_project_config
+NEW ir.ui.menu: project.menu_project_tags_act
+NEW ir.ui.menu: project.menu_projects_config
+NEW ir.ui.menu: project.portal_services_projects
+DEL ir.ui.menu: base.menu_definitions
+DEL ir.ui.menu: base.menu_project_config
+DEL ir.ui.menu: project.menu_project_category_act
+DEL ir.ui.menu: project.menu_task_types_view
+NEW ir.ui.view: project.project_planner
+NEW ir.ui.view: project.project_tags_form_view
+NEW ir.ui.view: project.project_tags_search_view
+NEW ir.ui.view: project.view_project_task_pivot
+NEW ir.ui.view: project.view_task_history_pivot
+NEW ir.ui.view: project.view_task_project_user_pivot
+DEL ir.ui.view: project.project_category_form_view
+DEL ir.ui.view: project.project_category_search_view
+DEL ir.ui.view: project.view_project_project_gantt
+DEL ir.ui.view: project.view_project_task_delegate
+DEL ir.ui.view: project.view_task_gantt
+DEL ir.ui.view: project.view_task_work_form
+DEL ir.ui.view: project.view_task_work_tree
+# ir.ui.* ---> Nothing to do
+
+NEW mail.message: project.msg_task_4
+NEW mail.message: project.msg_task_data_14
+NEW mail.message: project.msg_task_data_8
+NEW mail.message.subtype: project.mt_project_task_ready
+DEL mail.message.subtype: project.mt_project_task_assigned
+DEL mail.message.subtype: project.mt_task_assigned
+# Related to mail module changes ---> Nothing to do
+
+NEW project.project: project.project_project_data
+NEW project.tags: project.project_tag_data
+NEW project.task: project.project_task_data_0
+NEW project.task: project.project_task_data_1
+NEW project.task: project.project_task_data_11
+NEW project.task: project.project_task_data_12
+NEW project.task: project.project_task_data_13
+NEW project.task: project.project_task_data_14
+NEW project.task: project.project_task_data_2
+NEW project.task: project.project_task_data_4
+NEW project.task: project.project_task_data_5
+NEW project.task: project.project_task_data_6
+NEW project.task: project.project_task_data_7
+NEW project.task: project.project_task_data_8
+NEW project.task: project.project_task_data_9
+NEW project.task.type: project.project_stage_data_0
+NEW project.task.type: project.project_stage_data_1
+NEW project.task.type: project.project_stage_data_2
+DEL project.task.type: project.project_tt_analysis
+DEL project.task.type: project.project_tt_cancel
+DEL project.task.type: project.project_tt_deployment
+DEL project.task.type: project.project_tt_design
+DEL project.task.type: project.project_tt_development
+DEL project.task.type: project.project_tt_merge
+DEL project.task.type: project.project_tt_specification
+DEL project.task.type: project.project_tt_testing
+# TODO - project and task related stuff - analysis needed
+
+DEL res.groups: project.group_delegate_task
+# TODO - res.groups - has it moved in another module? - analysis needed
```

Comment:
Delegation has been 'simplified' in Odoo 9.0 ;-) https://www.odoo.com/groups/community-59/community-14009273 -> Nothing to do.